azurerm/internal/services/appplatform/spring_cloud_service_resource.go

```diff
@@ -422,6 +480,17 @@ func expandArmSpringCloudGitPatternRepository(input []interface{}) (*[]appplatfo
 	return &results, nil
 }
 
+func expandArmSpringCloudTrace(input []interface{}) *appplatform.TraceProperties {
+	if len(input) == 0 || input[0] == nil {
+		return nil
+	}
+	v := input[0].(map[string]interface{})
+	return &appplatform.TraceProperties{
+		Enabled:                      utils.Bool(v["enabled"].(bool)),
```

Comment:
since the presence of this block infers this is enabled - we can hard-code this:

```suggestion
		Enabled:                      utils.Bool(true),
```